rocAL/include/readers/image/image_reader.h

```diff
@@ -118,6 +120,7 @@ struct ReaderConfig {
     bool _shuffle = false;
     bool _loop = false;
     std::string _file_prefix = "";  //!< to read only files with prefix. supported only for cifar10_data_reader and tf_record_reader
+    std::string _file_list_path = "";  //!< to read only files present in the file list
```

Comment:
I think by default it will be created as an empty string. So initialization is not required. Same for _file_list_path (L#123) as well